.github/workflows/release-winget.yml

```diff
@@ -27,32 +29,59 @@ jobs:
 
       - name: Publish manifest with winget-create
         run: |
-          # Get correct release asset
-          $github = Get-Content '${{ github.event_path }}' | ConvertFrom-Json
```

Comment:
Right, this never could have worked with any webhook event other than `release`.